pkgs/by-name/vu/vulkan-hdr-layer-kwin6/package.nix

```
@@ -0,0 +1,50 @@
+{
+  lib,
+  stdenv,
+  cmake,
+  fetchFromGitHub,
+  libX11,
+  meson,
+  ninja,
+  pkg-config,
+  vulkan-headers,
+  vulkan-loader,
+  wayland-scanner,
+  wayland,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "vulkan-hdr-layer-kwin6";
+  version = "0-unstable-2024-10-19";
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    cmake
+  ];
+
+  buildInputs = [
+    vulkan-headers
+    vulkan-loader
+    libX11
+    wayland-scanner
+    wayland
+  ];
```

```suggestion
  depsBuildBuild = [ pkg-config ];

  nativeBuildInputs = [
    meson
    ninja
    pkg-config
    cmake
    wayland-scanner
  ];

  buildInputs = [
    vulkan-headers
    vulkan-loader
    libX11
    wayland
  ];

  strictDeps = true;
```
Fixes cross-compiling